.ci_scripts/update_docs

```diff
@@ -4,14 +4,14 @@
 set -ex
 
 if [ -n "$GH_TOKEN" ]; then
+    # TODO What is the purpose of this check? If we're on master, then check out master??
```

Comment:
If we're on a branch named master but not the latest master, then checkout latest master.

Reply:
In that situation, git will fail:

make sure we're up to date with upstream master:
```
$  git reset --hard origin/master
HEAD is now at 45211c7 Merge pull request #817 from scopatz/no-search                                                   
```
Clean up cruft from local ide's and such
```
$ git clean -xfd
Removing .idea/                  
Removing scripts/.watch_only_my_feedstocks.py.swp                                                                     
```
Check the status
```
$ git status
On branch master                                                                                                        
nothing to commit, working tree clean                               
```

Checkout the previous commit on master, then delete old master, then name our new branch master. This new master branch is one commit behind upstream master:
```
$ git checkout HEAD~
Note: checking out 'HEAD~'.
                  
You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.
                      
If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:
                                
  git checkout -b <new-branch-name> 
                             
HEAD is now at 825ebf6 Merge pull request #815 from ericdill/minutes

$ git branch -D master
Deleted branch master (was 45211c7).                          

$ git checkout -b master
Switched to a new branch 'master'                                              
```

Now, let's try this command that's in the .travis.yml file:
```
$ git checkout master
Already on 'master'
```
So, this "git checkout master" command has no effect because we're already on master. If we want to be on the latest master then it would have to be something like
```
$ git fetch origin
$ git reset --hard origin/master
$ git clean -xfd
```